geode-core/src/main/java/org/apache/geode/internal/cache/tier/MessageType.java

```diff
@@ -12,604 +12,430 @@
+  REGISTER_INTEREST_DATA_ERROR(21),
 
   /**
    * A message to unregister interest in a specific key
    *
    * @since GemFire 4.1.2
    */
-  public static final int UNREGISTER_INTEREST = 22;
+  UNREGISTER_INTEREST(22),
 
   /**
    * The unregister interest request data was bad (null region name, for example)
    *
    * @since GemFire 4.1.2
    */
-  public static final int UNREGISTER_INTEREST_DATA_ERROR = 23;
+  UNREGISTER_INTEREST_DATA_ERROR(23),
 
   /**
    * A message to register interest in a specific list of keys
    *
    * @since GemFire 4.1.2
    */
-  public static final int REGISTER_INTEREST_LIST = 24;
+  REGISTER_INTEREST_LIST(24),
 
   /**
    * A message to unregister interest in a specific list of keys
    *
    * @since GemFire 4.1.2
    */
-  public static final int UNREGISTER_INTEREST_LIST = 25;
+  UNREGISTER_INTEREST_LIST(25),
 
   /**
    * An unknown message type. This is being used for responses.
    *
    * @since GemFire 4.1.2
    */
-  public static final int UNKNOWN_MESSAGE_TYPE_ERROR = 26;
+  UNKNOWN_MESSAGE_TYPE_ERROR(26),
 
   /**
    * The receiver (which is an edge client in this case) should locally create a piece of data
    *
    * @since GemFire 4.1.2
    */
-  public static final int LOCAL_CREATE = 27;
+  LOCAL_CREATE(27),
 
   /**
    * The receiver (which is an edge client in this case) should locally update a piece of data
    *
    * @since GemFire 4.1.2
    */
-  public static final int LOCAL_UPDATE = 28;
+  LOCAL_UPDATE(28),
 
-  public static final int CREATE_REGION = 29;
-  public static final int CREATE_REGION_DATA_ERROR = 30;
+  CREATE_REGION(29),
+  CREATE_REGION_DATA_ERROR(30),
 
   /**
    * A message to make primary server
    *
    * @since GemFire 5.1
    */
-  public static final int MAKE_PRIMARY = 31;
+  MAKE_PRIMARY(31),
 
   /**
    * Response message type from primary server
    *
    * @since GemFire 5.1
    */
-  public static final int RESPONSE_FROM_PRIMARY = 32;
+  RESPONSE_FROM_PRIMARY(32),
 
   /**
    * Response message type from secondary server
    *
    * @since GemFire 5.1
    */
-  public static final int RESPONSE_FROM_SECONDARY = 33;
+  RESPONSE_FROM_SECONDARY(33),
 
   /**
    * A query request message
    *
-   * <p>
-   * author gregp
-   *
    * @since GemFire 4.1.1
    */
-  public static final int QUERY = 34;
+  QUERY(34),
 
-  public static final int QUERY_DATA_ERROR = 35;
+  QUERY_DATA_ERROR(35),
 
-  public static final int CLEAR_REGION = 36;
+  CLEAR_REGION(36),
 
-  public static final int CLEAR_REGION_DATA_ERROR = 37;
+  CLEAR_REGION_DATA_ERROR(37),
 
-  public static final int CONTAINS_KEY = 38;
+  CONTAINS_KEY(38),
 
-  public static final int CONTAINS_KEY_DATA_ERROR = 39;
+  CONTAINS_KEY_DATA_ERROR(39),
 
-  public static final int KEY_SET = 40;
+  KEY_SET(40),
 
-  public static final int KEY_SET_DATA_ERROR = 41;
+  KEY_SET_DATA_ERROR(41),
 
-  public static final int EXECUTECQ_MSG_TYPE = 42;
+  EXECUTECQ_MSG_TYPE(42),
```

Comment:
All of these CQ ones that end with _MSG_TYPE dropped "_MSG_TYPE" from the String that was returned from getString. Since you now just use the Enum's string, I think you should rename all these CQ ones to not have _MSG_TYPE in their name. That way any log messages will be the same as before.